api/src/main/java/org/openmrs/OrderGroup.java

```diff
@@ -35,6 +35,8 @@
 	
 	private OrderSet orderSet;
 	
+	private Concept parentOrderGroup;
```

Comment:
Shouldn't a parent order group be an OrderGroup?

Reply:
yes @dkayiwa i have it made an OrderGroup,kindly review